docs/Information-Flow.md

```diff
@@ -0,0 +1,88 @@
+# Information Flow
+
+It's recommended you understand the [basics of `sbp`](Style-Guide.md#sbp) before moving on because everything that will come next, works around it.
+
+The simplest way to explain this is by starting to explain a couple of functions you'll see together a lot of times across the project. These two functions are the core to make any logical change at Group Income:
+
+```js
+// Example: Update a group minimum income to 150
+
+// 1. Message - Create a set of data to be later processed
+const message = await sbp(
+  'gi.contracts/group/updateSettings/create',
+  { incomeProvided: 150 },
+  currentGroupId
+)
+
+// 2. Mutation - Send the message to the server and update the store (Vuex) accordingly
+await sbp('backend/publishLogEntry', message)
+```
+
+Now, let me explain in more detail what really happens in each function:
+
+##### 1. Create a Message
+
+The SBP selector [_domain_ `gi.contracts/`](../frontend/model/contracts/group.js) is based on a [common Contract](../frontend/model/Contract.js). Each action (ex: `group/updateSettings`) expects 2 arguments: the `data` and `contractId`.
```

Comment:
This isn't quite right. Contract actions do not expect 2 arguments, they *take data*, which is of the type specified by `validate`.